### ISSUE TITLE
Add support for collections upload to take a file path

### ISSUE THREAD

**Issue by n-wach** [FLIR/conservator-cli#151]
Once this is done, we can consider deprecating the individual media upload commands.

Advantages of the individual media uploads are support for orphans, and support for specifying remote name.

Low priority tho, as there are easy, documented workarounds

**Comment by shiftycow** [comment]
https://jiracommercial.flir.com/browse/CON-1249

**Comment by n-wach** [comment]
We will track this internally via JIRA
